src/hooks/useReferencePriceSubscriber.ts

```diff
@@ -27,9 +27,13 @@ const useGasPriceSubscriber = () => {
     () => () => {
       if (intervals.current.gas) return;
       const updateGasPrice = async () => {
-        const price = await getFastGasPrice();
-        if (price) {
-          dispatch(setFastGasPrice(price.toString()));
+        try {
+          let price = await getFastGasPrice();
+          if (price) {
+            dispatch(setFastGasPrice(price.toString()));
+          }
+        } catch (e) {
+          console.error(e);
```

Comment:
How about adding an extra function with comment to make it a little bit more readable?

```
// Fix for call not working in dev
const getFastGasPriceFix = async (): Promise<string> => {
  try {
    const price = await getFastGasPrice();
    return price ? price.toString() : "0";
  } catch (e) {
    console.error(e);
    return "0";
  }
}

-----

const updateGasPrice = async () => {
 const price = await getFastGasPriceFix();
 if (parseFloat(price)!==0) {
    dispatch(setFastGasPrice(price));
 }
};
```

Reply:
Ok never mind!